test/code/exit.test

```diff
@@ -0,0 +1,11 @@
+<?php
+$a = 1;
+exit;
+echo $a;
+-----
+Block#1
+    Expr_Assign
+        var: Var#1<$a>
+        expr: LITERAL(1)
+        result: Var#2
+    Terminal_Exit
```

Comment:
Can you please also add a variant where exit has an argument? That is `exit($a);`